deploy-manage/cloud-connect.md

```diff
@@ -13,9 +13,9 @@ Cloud Connect enables you to use {{ecloud}} services in your self-managed cluste
 
 AutoOps is the first service available for use with Cloud Connect. More services are coming soon.
 
-### AutoOps
+### AutoOps for self-managed clusters
 
-[AutoOps](/deploy-manage/monitor/autoops.md) is a monitoring tool that helps you manage your cluster with real-time issue detection, performance recommendations, and resolution paths. By analyzing hundreds of {{es}} metrics, your configuration, and usage patterns, AutoOps recommends operational and monitoring insights that deliver real savings in administration time and hardware cost. 
+[AutoOps](/deploy-manage/monitor/autoops.md) is a monitoring tool that helps you manage your cluster with real-time issue detection, performance recommendations, and resolution paths. By analyzing hundreds of {{es}} metrics, your configuration, and your usage patterns, AutoOps recommends operational and monitoring insights that deliver real savings in administration time and hardware cost. 
```

Comment:
please clarify the meaning of self-managed clusters in these faq (perhaps just in this opening section)